societies/swg.md

```diff
@@ -12,7 +12,7 @@ The Student Mentorship Programme is the flagship initiative of SWG which is held
 - Enabling constructive and positive interaction, guidance and mentorship of junior students by senior students. 
 - Providing a reliable and comprehensive support system from within and for the student community to motivate students to excel in both academic and non-academic fields and to make the most of their life at IIT Kharagpur. 
 
-Governors: <br />
```

Comment:
Could you remove the image from this file. We will add it again when we have some.